vcd/datasource_vcd_nsxt_ip_discovery_profile.go

```diff
@@ -0,0 +1,152 @@
+package vcd
```

Comment:
Given the name of the data source in `provider.go`, this file should be called `datasource_vcd_nsxt_segment_ip_discovery_profile.go`

Reply:
Thanks. Changed all 5 filenames and function names.